components/kernels/kernel-page.js

```diff
@@ -18,9 +18,8 @@ export const kernels = [
 ];
 
 export default (props: KernelPageProps) => (
-  <Layout pageTitle=": The nteract Desktop App is cool">
+  <Layout pageTitle={`: kernels - ${props.language}`}>
     <Head>
-      <title>{`kernels - ${props.language}`}</title>
       <link rel="stylesheet" href="/static/kernels.css" />
```

Comment:
Side note about next's `<Head>` -- these get appended to any prior `<Head>` entries. 😄